src/main/java/core/basesyntax/service/impl/WriteToFileImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriteToFile;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class WriteToFileImpl implements WriteToFile {
+    @Override
+    public boolean writeToFile(String filePath, String report) {
+        File file = new File(filePath);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file " + filePath + e);
+        }
+        try {
+            Files.write(file.toPath(), report.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file " + filePath + e);
```

Comment:
```suggestion
            throw new RuntimeException("Can't write to file " + filePath , e);
```
here same, maybe better ask mentor if this ok:)